index.html

```diff
@@ -9,11 +9,12 @@
   <body>
     <div>
       <header>
-        <h2 class="studentName">Student Page</h2>
+        <h2 class="studentName">Meera Iyad</h2>
       </header>
 
       <section class="container">
-        <h3 class="aboutMe">About me</h3>
+        <h3 class="aboutMe">Hello, my name is Meera. I am studying computer science  at Palestine polyticnic university.  I am 19. I am passionate about solving problems in general. I am a minimalist person 
+          ( I believe that less stuff can help us to be happier and less stressed).</h3>
```

Comment:
Great Job!! :partying_face: 
Just notice that this section has to have a header, which is **About me** in our case,
so make sure to add this description on a `<p>` tag not an `<h3>`, since this long paragraph shouldn't be a header as you may notice :eyes: 
for example:
```html
<section class="container">
    <h3 class="aboutMe">About me</h3>
    <p>your paragraph</p>
</section>
```